File: backend/src/VirtualScale.Api/Dtos/ScaleResponse.cs
namespace VirtualScale.Api.Dtos;

public record ScaleResponse(
    decimal BruteWeight,
    decimal NetWeight,
    decimal TareWeight,
    bool IsTared,
    bool IsStable,
    int FilterLevel,
    int NumberOfCells,
    decimal CapMax,
    int Division,
    int DecimalPlaces,
    decimal ReferenceWeight,
    bool NeedsCalibrationAdjustment
);

File: frontend/src/app/features/scale/models/scale-response.ts
export interface ScaleResponse {
  bruteWeight: number;
  netWeight: number;
  tareWeight: number;
  isTared: boolean;
  isStable: boolean;
  filterLevel: number;
  numberOfCells: number;
  capMax: number;
  division: number;
  decimalPlaces: number;
  referenceWeight: number;
  resolution: number;
  unit: string;
  needsCalibrationAdjustment: boolean;
}
